test/gtest/uct/ib/test_dc.cc

```diff
@@ -531,11 +540,15 @@ class test_dc_flow_control : public test_rc_flow_control {
         }
     }
 
-    virtual void test_pending_grant(int16_t wnd)
+    virtual void test_pending_grant(int16_t wnd) override
     {
         test_rc_flow_control::test_pending_grant(wnd);
         flush();
     }
+
+    virtual std::string wrap_config_str(const std::string &str) const override {
```

Comment:
why need wrap_config_str? it seems hacky.. i'd expect a config without transport name to work..

Reply:
let's see if it might be fixed in config (https://github.com/openucx/ucx/pull/10057)